src/main/java/com/lpvs/service/scanner/scanoss/LPVSScanossDetectService.java

```diff
@@ -69,15 +79,17 @@ public void runScan(LPVSQueue webhookConfig, String path) throws Exception {
             } else {
                 log.error("Directory %s could not be created." + resultsDir.getAbsolutePath());
             }
-            processBuilder =
-                    new ProcessBuilder(
-                            "scanoss-py",
-                            "scan",
-                            debug ? "-t" : "-q",
-                            "--no-wfp-output",
-                            "-o",
-                            getScanResultsJsonFilePath(webhookConfig),
-                            path);
+            if (processBuilder == null) {
```

Comment:
I'm not sure if this logic is appropriate. What if processBuilder is not null along with the other pervious webhookConfig information?

Reply:
Previous version of LPVSScanossDetectService is not testable at this point, which indicates the presence  of issues in architecture. That's why I have to make changes in the declaration of ProcessBuilder to make it mockable. LPVSScanossDetectService uses only scanoss scanner, so it doesn't depend on webhookconfig.

Regarding the update in GitHub actions, I make them to cover more lines and branches by tests, but they can probably be removed from current PR if needed.